### ISSUE TITLE
It's impossible to type/enter input on mobile devices

### ISSUE THREAD

**Issue by alexkrys123** [jcubic/jquery.terminal#846]
### Issue summary
It's impossible to type/enter input on mobile devices - no keyboard appears to type input

### Expected behavior
Keyboard should appear to type on it

### Actual behavior
Nothing appears to type

### Steps to reproduce
Try on https://terminal.jcubic.pl/ or https://terminal.jcubic.pl/prism.html - none of the sites provide an option to input the text

### Browser and OS
Chrome, Opera on Android

**Comment by jcubic** [comment]
Thanks for the report, it seems that the latest version broke mobile support. The keyboard doesn't work. If you need this ASAP you can use version 2.34.0 You can test it here:

https://jcubic.github.io/jquery.terminal/?ver=2.34.0

**Comment by redthing1** [comment]
I tried that previously version on Android and it does not work either. I have a Google Pixel XL on Android 11.

**Comment by redthing1** [comment]
It works in mobile chrome but not Firefox.

**Comment by jcubic** [comment]
Will check. It may take a while to fix.

**Comment by redthing1** [comment]
Thanks. Really appreciate, ping me whenever you get to it.

**Comment by adamclement-exe** [comment]
tested my site on android chrome with v2.34 today and is working http://adamsbashgame.pythonanywhere.com/

not with latest version though

**Comment by jcubic** [comment]
Firefox is fixed on the devel branch, I've added something that was supposed to work but I didn't actually test how it would work on Firefox. And there are automated tests like this.

I still need to figure out what change in version 2.35 make mobile Chrome fail to open the virtual keyboard.

**Comment by jcubic** [comment]
I've found that the problem was:
```css
.terminal, .terminal *, .cmd, .cmd *, .cmd textarea {
    all: revert;
}
```
CSS added in 2.35, this is a bug in Chrome that I've reported here to the Chromium project:

[CSS rule `* { all: revert; }` breaks contenteditable on mobile](https://bugs.chromium.org/p/chromium/issues/detail?id=1403492).

**Comment by jcubic** [comment]
The fix was released in version 2.35.1.
